docs/website/content/v0.6/en/guides/local/registry-cache.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Registry Cache
+---
+
+In this guide we will create a set of local caching Docker registry proxies to minimize local cluster startup time.
+
+When running Talos locally, pulling images from Docker registries might take significant amount of time.
+We spin up local  caching pass-through registries to cache images and configure local Talos cluster to use those proxies.
+Similar approach might be used to run Talos in production in air-gapped environments.
+It can be also used to verify that all the images are available in local registries.
+
+## Requirements
+
+The follow are requirements for creating set of caching proxies:
+
+- Docker 18.03 or greater
+- Local cluster requirements for either [docker](docker) or [fireracker](firecracker).
+
+## Launch the Caching Docker Registry Proxies
+
+With default configuration, Talos pulls from `docker.io`, `k8s.gcr.io` and `quay.io`.
+If your configuration is different, you might need to modify the commands below:
+
+```bash
+docker run -d -p 5000:5000 \
+    -e REGISTRY_PROXY_REMOTEURL=https://registry-1.docker.io \
+    --restart always \
+    --name registry-docker.io registry:2
+
+docker run -d -p 5001:5000 \
+    -e REGISTRY_PROXY_REMOTEURL=https://k8s.gcr.io \
+    --restart always \
+    --name registry-k8s.gcr.io registry:2
+
+docker run -d -p 5002:5000 \
+    -e REGISTRY_PROXY_REMOTEURL=https://quay.io \
+    --restart always \
+    --name registry-quay.io registry:2.5
+```
+
+> Note: Proxies are started as docker containers, and they're automatically configured to start with Docker daemon.
+Please note that `quay.io` proxy doesn't support recent Docker image schema, so we run older registry image version (2.5).
+
+As a registry container can only handle single upstream Docker registry, we launch a container per upstream, each on its own
+host port (5000, 5001, 5002).
```

Comment:
I think the linter may hit you on this since the sentence is not on a single line.

Reply:
I think linter makes me start each sentence on a new line, but it doesn't force whole sentence to be on a single line.